src/ape/utils.py

```diff
@@ -31,12 +31,20 @@
     from functools import cached_property  # type: ignore
 except ImportError:
     from backports.cached_property import cached_property  # type: ignore
+"""
```

Comment:
In general, don't worry about documenting imported stuff.

Don't worry about this - the work I am doing will automatically grab the docs strs from the imported property if we are including it in `__all__` .

Reply:
okay thank you